lib/versions.js

```diff
@@ -1,7 +1,10 @@
 const semver = require('semver')
 
 const splitSemVer = input => {
-  const version = semver.inc(input.lastVersion, input.inc, true)
+  const version =
+    !input.increment && input.inc === 'patch'
```

Comment:
I think it's pretty clever to not increment the patch version when providing a version or tag with an input. But I feel like it may not be as intuitive for our users. What about adding a new template variable: `$INPUT_VERSION` and leaving the behavior of the patch version like it was before (e.g. if input version is `v2.2.1`, `$NEXT_PATCH_VERSION` will become `v2.2.2` and `$INPUT_VERSION` will be `v2.2.1`).

Reply:
Yeah, good idea. I'll change that.